enhancements/monitoring/user-workload-monitoring.md

```diff
@@ -386,13 +380,12 @@ As a first step a similar high cardinality alert will be used as in [9] and pote
 
 The above will be encapsulated in one single alert.
 This alert will be declared as `info` level only and transmitted via Telemetry.
+
 Additional telemetry metrics are going to be gathered to have more insights about the usage of user workload monitoring in clusters:
 
 - Active series
 - Rate of sample ingested
 - Memory usage
-- CPU usage of prometheus
```

Comment:
We did not think CPU usage of prometheus is a problem right now, so we are not interested in tracking this at the moment.